sandbox-templates/remark-rehype-with-parcel/src/index.js

```diff
@@ -0,0 +1,34 @@
+// ⚠️ Important! Please make sure the dependencies are up to date.
+// You can refresh them in the Dependencies section (left-bottom on CodeSandbox)
```

Comment:
with stackblitz also being linked should this comment be updated?
suggestions on new copy?